### PR TITLE
CCIP-2929 Generate MCMS gethwrappers in ccip-owner-contracts

### PR DESCRIPTION
**Pull request by cmalec**

### CODE REVIEW

**Review by akhilchainani — COMMENTED** [review]
Looks fine, overall. I don't think we need all of the `.abi` and `.bin` files, though?

**Review by akhilchainani — COMMENTED** [review]
Also should we be committing the `abigen` binary?

File: GNUmakefile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.PHONY: abigen
+abigen: ## Build & install abigen. Location /tools/bin/
+	./gethwrappers/generation/build_abigen
```
Comment (connorwstein) [inline GNUmakefile:3]: can we pin the abigen version as well? Some helpful code might be https://github.com/smartcontractkit/ccip/blob/83ca5683607b9cc419891e7ede4867b467f5d4b3/tools/bin/build_abigen

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Contract code
+/src @smartcontractkit/research @smartcontractkit/security
```
Comment (cmalec) [inline CODEOWNERS:2]: @connorwstein seems we would need to ask sec to give these teams write access as a whole first so removing for now and sec tickets takes days to get done. i will add it after

File: GNUmakefile (hunk 1 above)
Comment (cmalec) [reply GNUmakefile:3]: already using that build scipt from core

**Review by akhilchainani — APPROVED** [review]

**Comment by cmalec** [comment]
https://github.com/smartcontractkit/ccip-owner-contracts/pull/3

**Pull request closed without merging**